config/global.ini.default

```diff
@@ -0,0 +1,25 @@
+[Database]
+Host = 127.0.0.1
+Port = 3306
+Database = sapphire
+Username = sapphire
+Password =
+SyncThreads = 2
+AsyncThreads = 2
+
+[Parameters]
+ServerSecret = default
+DataPath = C:\\SquareEnix\\FINAL FANTASY XIV - A Realm Reborn\\game\\sqpack
+WorldID = 67
```

Comment:
this should be in world

Reply:
it's needed for lobby/world

Reply:
Ah, fair enough. We shall revisit it if we decide to allow multiple worlds.

Reply:
yeah, I only moved it into global because we using it heaps in housing and we've hardcoded it everywhere

Reply:
speaking of which, going to rename this to general